### PR TITLE
feat/docs: add API docs for users and groups, improve README and commandline docs

### PR DESCRIPTION
**Pull request by rudyardrichter**
### New Features

- API documentation on users/groups

### Improvements

- README startup guide
- Commandline argument documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,56 @@ microservices needing to check user authorization to operate on a resource can
...
+
+You will need these:
+
+- [Go](https://golang.org/dl/)
+- [PostgreSQL](https://www.postgresql.org/download/)
+
+```bash
+go get -u github.com/uc-cdis/arborist
+make
+psql
```
Comment (fantix) [inline README.md]: I'm a bit confused about the `psql` here

File: README.md [hunk 2]
```diff
@@ -21,6 +21,56 @@ microservices needing to check user authorization to operate on a resource can
...
+export PGDATABASE=arborist_test PGUSER=username PGHOST=localhost PGPORT=5432 PGSSLMODE=disable
+# export any other PG variables as necessary
+./migrations/latest
```
Comment (fantix) [inline README.md]: ditto

File: docs/openapi.yaml [hunk 3]
```diff
@@ -85,21 +85,19 @@ paths:
         401:
           description: >-
             Token failed to validate (authentication error)
-  /engine/:
```
Comment (fantix) [inline docs/openapi.yaml]: Maybe also remove the "engine" at line 12~17?

File: README.md (hunk 1 above)
Comment (rudyardrichter) [reply README.md]: uh oh, this part is not completed. will fix

File: README.md (hunk 2 above)
Comment (rudyardrichter) [reply README.md]: ditto, have to fix

**Pull request merged**